run.py

```diff
@@ -21,6 +32,13 @@
 monikers_flat = [monikers_tuple[i][j] for i in range(len(monikers_tuple)) for j in range(3)]
 monikers_str  = '\n'.join(f"{i[0]}: {i[2]} doge" for i in monikers_tuple)
 
+md = (
+		MarkdownIt()
+		.use(front_matter_plugin)
+		.disable('image')
+		.enable('table')
```

Comment:
```suggestion
```

these aren't necessary for the markdown parser. Also the imports of the extensions should also be removed in this file.